### ISSUE TITLE
Not showing correct number of deprecations on the status bar

### ISSUE THREAD

**Issue by 50Wliu** [atom/deprecation-cop#37]
![image](https://cloud.githubusercontent.com/assets/2766036/7243636/a4b268a2-e79d-11e4-8976-2e3650594e3c.png)
but
![image](https://cloud.githubusercontent.com/assets/2766036/7243639/af0a017a-e79d-11e4-9f97-e75fd08e02d1.png)
0.193.0-7a9b167 on Ubuntu 14.10 with the following packages installed:
```
├── file-icons@1.5.4
├── fizzier@0.0.0
├── fizzy@0.15.0
├── git-control@0.1.9
├── isotope-light-ui@2.0.2
├── jenkins@0.3.2
├── language-sourcepawn@0.5.0
├── linter@0.12.0
├── linter-jshint@0.1.2
├── one-light-syntax@0.4.0
├── one-light-ui@0.5.3
├── sourcepawn-compiler@0.0.0
├── tabs-to-spaces@0.9.2
└── travis-ci-status@0.13.0
```

**Comment by izuzak** [comment]
I was able to reproduce this with the git-control package. Here's what I'm seeing (15 vs 7):

![screen shot 2015-04-21 at 09 33 00](https://cloud.githubusercontent.com/assets/38924/7247407/6e87302a-e809-11e4-8cce-a6cf396144e7.png)

Steps to reproduce:
1. install git-control package
2. run the Git Control: Toggle command from command palette

**Comment by chemoish** [comment]
I am having the same issues, but it looks like its not the case for everyone (As people are sending me the deprecation messages).

https://github.com/chemoish/atom-pane-layout/issues/10

Windows 7
Atom 0.194.0

TLDR — The warning count updates for me upon package activation (hotkey), but no messages show up in the window.

**Comment by izuzak** [comment]
Okay, so this happens because the status bar icon [counts deprecations as they are received from Grim](https://github.com/atom/deprecation-cop/blob/daae43f77dcd124760d1566ad1476e979a878d5f/lib/deprecation-cop-status-bar-view.coffee#L43). However, a single Grim deprecation might actually be a group of multiple deprecations with different call stacks. The Deprecation Cop view [un-groups those](https://github.com/atom/deprecation-cop/blob/daae43f77dcd124760d1566ad1476e979a878d5f/lib/deprecation-cop-view.coffee#L155-L162) so that all of them can be shown to the user. 

So, we need to make those two ways of counting deprecations match.
